form/without_class.rst

```diff
@@ -123,3 +126,56 @@ but here's a short example::
     When a form is only partially submitted (for example, in an HTTP PATCH
     request), only the constraints from the submitted form fields will be
     evaluated.
+
+Constraints At Class Level
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Another possibility is to add the constraints at the class level.
+This can be done by setting the ``constraints`` option in the
+``configureOptions()`` method::
+
+    use Symfony\Component\Form\Extension\Core\Type\TextType;
+    use Symfony\Component\Form\FormBuilderInterface;
+    use Symfony\Component\Validator\Constraints\Length;
+    use Symfony\Component\Validator\Constraints\NotBlank;
```

Comment:
Ue statement for OptionsResolver is missing

Reply:
Good catch 👍